tests/runtime/usdt

```diff
@@ -74,6 +74,10 @@ RUN {{BPFTRACE}} -e 'usdt:*:tracetest:testprobe { printf("here\n" ); exit(); }'
 EXPECT here
 BEFORE ./testprogs/usdt_test
 NEW_PIDNS
+# On 9pfs, is_elf() hangs on a syscall to fstat64. We suspect it's
```

Comment:
nit: should we make an issue to re-enable?

Reply:
Probably not necessary. When vmtest switches over to virtiofs this will automatically be re-enabled.

And I don't know that 9pfs has any plans to fix this. Nor do I plan on fixing it. So the issue probably doesn't serve any purpose.